### PR TITLE
Fix the Numbered Bullets in the Installing Ballerina Page

### LINKED ISSUE #232: Incorrect numbering in Installing Ballerina page
**Description:**

$subject in [1]

https://ballerina.io/learn/installing-ballerina/

image

image

**Suggested Labels:**

**Suggested Assignees:**

**Affected Product Version:**

**OS, Browser, other environment details and versions:**    

**Steps to reproduce:**

**Related Issues:**

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
This is to Fix the Numbered Bullets in the Installing Ballerina Page.
> Fixes #232 

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: learn/installing-ballerina.md [hunk 1]
```diff
@@ -72,20 +72,20 @@ Follow the steps below to obtain the Ballerina source code.
...
+  git clone --recursive https://github.com/ballerina-platform/ballerina-lang.git
+  ```
+  >**Tip:** If you have already forked the repository to your GitHub account, then execute the below command replacing <YOUR-GITHUB-USERNAME> with your Git username.
 
-```
-git clone --recursive https://github.com/<YOUR-GITHUB-USERNAME>/ballerina-lang.git
-```
+  ```
+  git clone --recursive https://github.com/<YOUR-GITHUB-USERNAME>/ballerina-lang.git
+  ```
```
Comment (anupama-pathirage) [inline learn/installing-ballerina.md]: Can we add one more tab to the sections under each numbered item to make that part more clear?

Comment (praneesha) [reply learn/installing-ballerina.md]: Done

File: learn/installing-ballerina.md [hunk 2]
```diff
@@ -61,9 +61,9 @@ Alternatively, follow the instructions below to install Ballerina from the sourc
 
 You need to download and install the below to build the Ballerina modules.
 1. Java SE Development Kit (JDK) version 8 (from one of the following locations) 
-- [Oracle](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
-- [OpenJDK](http://openjdk.java.net/install/index.html)
->**Note:** Set the JAVA_HOME environment variable to the path name of the directory into which you installed JDK.
+    - [Oracle](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
+    - [OpenJDK](http://openjdk.java.net/install/index.html)
+  >**Note:** Set the JAVA_HOME environment variable to the path name of the directory into which you installed JDK.
```
Comment (anupama-pathirage) [inline learn/installing-ballerina.md]: This is not intended.

File: v0-991/learn/getting-started.md [hunk 3]
```diff
@@ -86,9 +86,9 @@ Alternatively, follow the instructions below to install Ballerina from the sourc
 
 You need to download and install the below to build the Ballerina modules.
 1. Java SE Development Kit (JDK) version 8 (from one of the following locations) 
-- [Oracle](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
-- [OpenJDK](http://openjdk.java.net/install/index.html)
->**Note:** Set the JAVA_HOME environment variable to the path name of the directory into which you installed JDK.
+    - [Oracle](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
+    - [OpenJDK](http://openjdk.java.net/install/index.html)
+  >**Note:** Set the JAVA_HOME environment variable to the path name of the directory into which you installed JDK.
```
Comment (anupama-pathirage) [inline v0-991/learn/getting-started.md]: This is not indented.

File: v1-1/learn/installing-ballerina.md [hunk 4]
```diff
@@ -60,9 +60,9 @@ Alternatively, follow the instructions below to install Ballerina from the sourc
 
 You need to download and install the below to build the Ballerina modules.
 1. Java SE Development Kit (JDK) version 8 (from one of the following locations) 
-- [Oracle](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
-- [OpenJDK](http://openjdk.java.net/install/index.html)
->**Note:** Set the JAVA_HOME environment variable to the path name of the directory into which you installed JDK.
+    - [Oracle](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
+    - [OpenJDK](http://openjdk.java.net/install/index.html)
+  >**Note:** Set the JAVA_HOME environment variable to the path name of the directory into which you installed JDK.
```
Comment (anupama-pathirage) [inline v1-1/learn/installing-ballerina.md]: This is not indented.

File: learn/installing-ballerina.md (hunk 2 above)
Comment (praneesha) [reply learn/installing-ballerina.md]: Fixed.

File: v0-991/learn/getting-started.md (hunk 3 above)
Comment (praneesha) [reply v0-991/learn/getting-started.md]: Fixed.

File: v1-1/learn/installing-ballerina.md (hunk 4 above)
Comment (praneesha) [reply v1-1/learn/installing-ballerina.md]: Fixed.

**Pull request merged**